### PR TITLE
added base structure for Evalai UI

### PR DESCRIPTION
**Pull request by aka-jain**
@dexter1691 Have a look and merge it if all OK.

### CODE REVIEW

File: frontend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# files/directories to be ignored
+
+.sass-cache/
+
+node_modules/
+bower_components/
```
Comment (deshraj) [inline frontend/.gitignore]: @aka-jain please add a new line here.

Comment (aka-jain) [inline frontend/.gitignore]: Sure

File: frontend/app.scripts.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+       "angular-animate" : "bower_components/angular-animate/angular-animate.min.js",
+       "angular-ui-router" : "bower_components/angular-ui-router/release/angular-ui-router.min.js",
+       "angular-aria" : "bower_components/angular-aria/angular-aria.min.js",
+       "angular-message": "bower_components/angular-messages/angular-messages.min.js",
+       "angular-material" : "bower_components/angular-material/angular-material.min.js"
+
+      }
+}
+
+
```
Comment (dexter1691) [inline frontend/app.scripts.json]: *nit*: Remove extra lines here please.

File: frontend/app.styles.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      "angular-material"
+
+
+    ]
+  },
+  "paths": {
+       
+       "animate": "bower_components/animate.css/animate.css",
+       "font-awesome":"bower_components/font-awesome/css/font-awesome.css",
+       "angular-material" : "bower_components/angular-material/angular-material.css"
```
Comment (dexter1691) [inline frontend/app.styles.json]: *nit*: can you put a comma at the end, and remove extra lines

**Comment by dexter1691** [comment]
Overall, looks good! There are minor aesthetic changes requested. Like eliminating extra lines throughout the code and putting commas at the end of last element in list.

**Comment by aka-jain** [comment]
Sure, working on it.

**Comment by aka-jain** [comment]
Done all changes. But I can't put comma at the end of list in app.scripts.json and app.styles.json because it's not a valid syntax and is throwing the compiling errors.

**Pull request merged**